src/Microsoft.Health.Dicom.Core.UnitTests/Features/Store/DicomDatasetValidatorTests.cs

```diff
@@ -43,7 +45,7 @@ public DicomDatasetValidatorTests()
         [Fact]
         public void GivenAValidDicomDataset_WhenValidated_ThenItShouldSucceed()
         {
-            _dicomDatasetValidator.Validate(_dicomDataset, requiredStudyInstanceUid: null);
+            _dicomDatasetValidator.Validate(_dicomDataset, new List<CustomTagEntry>(), requiredStudyInstanceUid: null);
```

Comment:
nit: `T[]` implements `IList<T>` and `IReadOnlyList<T>`, so you can pass `Array.Empty<CustomTagEntry>()` here and below where you need some empty list of stuff (just don't try adding to it).

I recommend this especially outside of test code as `Empty` doesn't allocate a new array.